yarn.lock

```diff
@@ -3892,26 +3978,27 @@
     "@radix-ui/react-id" "0.1.5"
     "@radix-ui/react-primitive" "0.1.4"
 
-"@radix-ui/react-popover@^0.1.6":
-  version "0.1.6"
-  resolved "https://registry.yarnpkg.com/@radix-ui/react-popover/-/react-popover-0.1.6.tgz#788e969239d9c55239678e615ab591b6b7ba5cdc"
-  integrity sha512-zQzgUqW4RQDb0ItAL1xNW4K4olUrkfV3jeEPs9rG+nsDQurO+W9TT+YZ9H1mmgAJqlthyv1sBRZGdBm4YjtD6Q==
+"@radix-ui/react-popover@^1.0.0":
```

Comment:
`yarn.lock` has received more changes than I think you anticipated, and some of these upgrades are causing some breaking UI changes.  I'm seeing some UI errors in popovers (see attached video).  There could be more issues.

I do think that we should upgrade these Radix components, but it is out of scope of this ticket.... Is upgrading radix popover from `^0.1.6` to `^1.0.0` critical for this ticket?

@crabsinger @CryptoGraffe What is the best way for @sshmm to reset his `yarn.lock` so that it matches main?

Popover UI alignment error:

https://user-images.githubusercontent.com/100873710/187508907-93d0d62c-fd65-469d-be1b-83b1aaaba018.mov

Reply:
Agree - I think easiest way to reset yarn.lock is
1. `git checkout main`
2. cp yarn.lock yarn.lock.main
3. git checkout `your_branch`
4. cp yarn.lock.main yarn.lock
5. done

@sshmm @wingfeatherwave

Reply:
@wingfeatherwave could you explain the error? is it the delay?

Reply:
The popover error is related to positioning and avatar sizing.  The avatar should be the same size and location when the popover is opened.  You can also see the position bug on the overview menu (the circle name doesn't line up)

I'm concerned that there could be more issues than just this popover bug as so many of the radix components changed with your yarn.lock.  That's why I'm recommending getting the yarn lock from main and postponing upgrading the components to another ticket.

Reply:
@sshmm Is upgrading radix popover from ^0.1.6 to ^1.0.0 critical for this ticket?

Reply:
@wingfeatherwave  I can't remember the exact reason. I will revert it and try to make it work without this update

Reply:
@wingfeatherwave I reverted it back. please recheck the functionality